background.js

```diff
@@ -188,6 +188,10 @@ async function load_accounts() {
         await waitFor(() => {
             return graph_api_token !== null;
         });
+        if ("error" in graph_api_token) {
+            ssoLog("couldn't aquire API token for avatar: " + graph_api_token.error)
```

Comment:
Aren't we logging errors twice now?

Reply:
> linux-entra-sso.py will send "Broker not available" as catch-all

That's true as we unfortunately cannot catch the GIO (Dbus) errors precisely (at least not without a lot of code). Under the assumption that we don't have any internal errors (which is of course questionable...), the only error that can happen is a broken communication with the broker. How about "Failure during processing of request"?

Reply:
> That's true as we unfortunately cannot catch the GIO (Dbus) errors precisely (at least not without a lot of code). Under the assumption that we don't have any internal errors (which is of course questionable...), the only error that can happen is a broken communication with the broker. 

We also rely on the message structure at several places (i.e. certain attributes being available), so an exception could as well mean that message format has changed or that the broker didn't like our request and responded with an error. 

> How about "Failure during processing of request"?

I think that covers all cases and I will update my PR to include this change later today.

Reply:
PR updated, solving both discussions, so from my side, it should be ready for merging, but let me know if there's anything else I should improve!